java/maven/src/org/netbeans/modules/maven/options/MavenSettings.java

```diff
@@ -480,6 +484,19 @@ public void setBinaryDownloadStrategy(DownloadStrategy ds) {
             }
         }
 
+        Pattern mavenCoreJarPattern = Pattern.compile("maven-core-(\\d+\\.\\d+\\.\\d+)\\.jar");  // NOI18N
```

Comment:
could be further extracted to a static final field (sorry that I didn't suggest it right away).

Reply:
👍 done